src/chrome/content/rules/CDN_Instagram.com.xml

```diff
@@ -6,6 +6,7 @@
 
 	<target host="scontent.cdninstagram.com" />
 	<target host="scontent-a.cdninstagram.com" />
+	<target host="scontent-ams3-1.cdninstagram.com" />
```

Comment:
Can you add a test url such as https://scontent-ams3-1.cdninstagram.com/vp/8928cde19409ea55ef8436fc5059bec6/5BC1D49D/t51.2885-15/e35/28434989_168324353967560_5452806340592271360_n.jpg please?